docs/HowtoCreateCollectionsOrRelatedLinksInMetalsmith.md

```diff
@@ -0,0 +1,104 @@
+# Adding a collection of related links
+_Added: 2017-04-27_
+
+Our current publishing system, Metalsmith, provides a way to group related content using what it calls _collections_. Using collections requires the `metalsmith-collections` plugin ([documentation](https://github.com/segmentio/metalsmith-collections)).
+
+Content grouped within a collection is displayed in the sidebar navigation. Viewing `content/components/navigation-sidebar.html` may clarify some of what's in this document.
+
+## Add a new collection
+
+_**NOTE**: Content team, you may wish to file a request with the DevOps team for this task._
+
+To create a new collection, add a new collection object to `vets-website/script/build.js`.
+
+
+```
+  smith.use(collections({
+    collectionName: {
+      pattern: 'path/to/the/files/*.md',
+      sortBy: 'order',
+      metadata: {
+        name: 'Title for collection 1'
+      }
+    },
+    secondCollectionName: {
+      pattern: 'path/to/the/files/*.md',
+      sortBy: 'order',
+      metadata: {
+        name: 'Title for collection two'
+      }
+    }
+  }));
+```
+
+### What each field means:
+
+- `collectionName`: should be a camel-cased or snake cased string that summarizes the name of the related documents.
+- `pattern`: should be a directory a relative to the site root (without a leading slash), and end with `*.md`.
```

Comment:
It'd helpful for non-technical folks to explain that the `/*.md` means the collection will include all of the markdown files within that folder.